pkg/cleaner/aws/aws.go

```diff
@@ -253,6 +255,17 @@ func stackShouldBeDeleted(stack *cloudformation.Stack) bool {
 	return false
 }
 
+func isGuestStack(stack *cloudformation.Stack) bool {
+	outputs := stack.Outputs
+	for _, o := range outputs {
+		if *o.OutputKey == "MasterImageID" {
+			return true
```

Comment:
It is the best way I found to identify it is a guest stack

Reply:
If it's easy to change can we make this `isTenantStack`? If not don't worry.